src/components/product/OutOfStock/OutOfStock.tsx

```diff
@@ -24,48 +26,94 @@ export interface OutOfStockProps {
   title?: string
   /**
    * Notification icon.
-   * @default <Icon name="Bell" />
+   * @default <Icon name="BellRinging" />
    */
   icon?: ReactElement
+  /**
+   * Notification icon.
+   * @default <Icon name="BellRinging" />
+   */
+  notificationIcon?: ReactElement
```

Comment:
Now that we have two icons, the props `icon` and `notificationIcon`, I'm imagining using this component from the outside, seeing its available props, and it's not immediatly clear to me what each icon is for, just by looking at their names. 

What do you think about renaming `icon` to `buttonIcon` (or `buttonInitialIcon`) and the other to `notificationMsgIcon`? Also in the documentation description, mentioning one is used in the message and the other in the button.

Reply:
Agreed  🙌🏽